AlphaWallet/Accounts/ViewModels/AccountsViewModel.swift

```diff
@@ -5,26 +5,46 @@ import Combine
 import Result
 import UIKit
```

Comment:
Looks like we can remove `import UIKit`. Great signal.

Reply:
**UIKit** is required there, but result can be deleted

Reply:
Hmm.. I thought it compiled when I removed it earlier.

Reply:
view model has `UISwipeActionsConfiguration` that comes from `UIKit`